src/cctp-sdk/index.ts

```diff
@@ -1,136 +1,78 @@
 import { ethers } from "ethers";
-import { jetch, poll } from "./helper";
-import { CIRCLE_API, GELATO_API, CHAIN } from "./constants";
+import { CONSTANTS, ChainId } from "./constants";
+import { AuthorizationStruct } from "../../typechain/contracts/GelatoCCTPSender";
 import {
-  CallWithSyncFeeERC2771Request,
-  CallWithSyncFeeRequest,
-  RelayResponse,
+  CallWithSyncFeeConcurrentERC2771Request,
   GelatoRelay,
 } from "@gelatonetwork/relay-sdk";
-import {
-  TaskState,
-  TaskStatus,
-  AttestationState,
-  AttestationStatus,
-  Authorization,
-} from "./types";
 import GelatoCCTPSenderAbi from "./abi/GelatoCCTPSender.json";
-import GelatoCCTPReceiverAbi from "./abi/GelatoCCTPReceiver.json";
 
 export const transfer = async (
   amount: bigint,
   srcMaxFee: bigint,
   dstMaxFee: bigint,
-  dstChainId: bigint,
+  srcChainId: ChainId,
+  dstChainId: ChainId,
   signer: ethers.Wallet
 ): Promise<void> => {
-  if (srcMaxFee + dstMaxFee > amount)
-    throw new Error("Max fee amount exceeds total amount");
-
-  const provider = signer.provider;
-  if (!provider) throw new Error("Signer missing provider");
-
-  const { chainId: srcChainId } = await provider.getNetwork();
   if (srcChainId === dstChainId)
     throw new Error("Source and destination chain must be different");
 
-  const srcChain = CHAIN[srcChainId.toString()];
-  const dstChain = CHAIN[dstChainId.toString()];
+  if (srcMaxFee + dstMaxFee > amount)
+    throw new Error("Max fee amount exceeds total amount");
 
-  if (!srcChain || !dstChain) throw new Error("Unsupported chain");
+  const src = CONSTANTS[srcChainId];
+  const dst = CONSTANTS[dstChainId];
 
   const srcAuthorization = await buildAuthorization(
     signer,
-    srcChain.usdc,
+    src.usdc,
     amount,
-    srcChain.sender,
+    src.gelatoSender,
```

Comment:
Always keep 1 name and 1 name only for Smart Contracts (and in general object instances) - so here please `gelatoCCTPSender` and same for receiver